pkg/rancher-desktop/config/settings.ts

```diff
@@ -251,17 +251,19 @@ export function load(deploymentProfiles: DeploymentProfileType): Settings {
       // 25% of available ram up to a maximum of 6gb
       settings.virtualMachine.memoryInGB = Math.min(6, Math.round(totalMemoryInGB / 4.0));
     }
-    if (os.platform() === 'linux' && !process.env['APPIMAGE']) {
+  }
+  if (os.platform() === 'linux' && !process.env['APPIMAGE']) {
```

Comment:
We need a comment specifying that it's ok to always disable `application.updater.enabled` on dev builds as well as CI runs, because that's what this does